features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/listener/WindowsOnDrawListenerTest.kt

```diff
@@ -102,12 +102,12 @@ internal class WindowsOnDrawListenerTest {
         }
         fakeWindowsSnapshots = fakeMockedDecorViews.map { forge.getForgery() }
         whenever(mockContext.theme).thenReturn(mockTheme)
-        fakeMockedDecorViews.forEachIndexed { index, decorView ->
+        fakeMockedDecorViews.forEachIndexed { index, _ ->
             whenever(
                 mockSnapshotProducer.produce(
-                    decorView,
-                    fakeSystemInformation,
-                    mockRecordedDataQueueRefs
+                    any(),
+                    any(),
+                    any()
```

Comment:
I would still keep the assertion of the arguments here:

```suggestion
                    eq(decorView),
                    eq(fakeSystemInformation),
                    any()
```